Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser.yml

```diff
@@ -9,7 +9,8 @@ configuration:
   name: email
   required: true
   type: 0
-- display: Auth Code (run the !gmail-auth-link command to start the auth flow - see Detailed Instructions (?) section)
+- display: Auth Code (run the !gmail-auth-link command to start the auth flow - see
+    Detailed Instructions (?) section)
```

Comment:
In 6.1 we eliminate the detailed instructions pop out. I'm okay with the wording, but it might not be relevant in the future.
![Screen Shot 2021-01-18 at 11 40 53](https://user-images.githubusercontent.com/42912128/104898390-413bf100-5982-11eb-8e4e-5fb27b73a8a3.png)

Reply:
We can leave for now. I think we have this across the board. At some time we will address this.